lib/sql/query-insert.js

```diff
@@ -51,7 +51,11 @@ module.exports = class Insert extends RowStreamQuery {
     for (let x = 0; x < preppedData.length; ++x) {
       const row = []
       for (const col of insertSchema) {
-        row.push('$' + this.values.push(preppedData[x][col]))
+        if (!(col in preppedData[x]) || (col === dao.primaryKeyName && !preppedData[x][col])) {
+          row.push('DEFAULT')
```

Comment:
This will do the `id::nextval` bit?

Reply:
yup, since that's the default in the schema this takes care of it